packages/bot/src/assets/sass/app.scss

```diff
@@ -1,2 +1,30 @@
 // Fonts
-@import 'deriv-shared/src/styles/google-fonts.scss';
\ No newline at end of file
+@import 'deriv-shared/src/styles/google-fonts.scss';
+
+.btn--icon {
```

Comment:
@sam-binary we dont do styles of components in app.scss. you need to create the proper class